src/standard/TIP-4/2_1.md

```diff
@@ -1,10 +1,10 @@
 ---
-title: 4.2. JSON Metadata
+title: 4.2. On-chain JSON Metadata
```

Comment:
You are adding a new On-chain JSON Metadata standard and attempting to replace the existing TIP-4.2 JSON Metadata, this is a standard; you can't just delete it, there are assets based on this standard that developers will need to work with, and you've decided to remove information about it, considering that the proposed changes are significant, the correct approach would be to add TIP-4.7 On-chain JSON Metadata

P.S. As for the broken build, check the build logs — all the answers are there

Reply:
see https://github.com/everscale-org/docs/pull/370/checks#step:4:109

Reply:
🙏 Additionally, I kindly ask you to pay attention to the change history formation, there is a [document that describes the process of making changes](https://github.com/everscale-org/docs/blob/main/CONTRIBUTING.md#main-flow), it provides all the necessary instructions, if necessary, just squash the commits so that all these attempts to make the changes don't clutter up the main project history — there's no point in that

Reply:
![image](https://user-images.githubusercontent.com/761285/159713949-34837e47-492b-4130-adf5-3941248de845.png)
see #13